src/styles/bloks/header.scss

```diff
@@ -0,0 +1,65 @@
+.header {
+  font-size: 12px;
+  font-weight: 500;
+
+  display: flex;
+  padding-left: 50px;
+  padding-right: 50px;
+  justify-content: space-between;
+  align-items: center;
+
+  &__logo-link {
+    align-content: center;
```

Comment:
The `align-content` property is not applicable to flex items themselves (such as your `&__logo-link`). This property is used with flex containers to align flex lines within it.